internal/app/cache/cache.go

```diff
@@ -219,14 +222,14 @@ func (c *cache) AddRequest(key string, req transport.Request) error {
 func (c *cache) DeleteRequest(key string, req transport.Request) error {
 	c.cacheMu.Lock()
 	defer c.cacheMu.Unlock()
-	metrics.CacheAddRequestSubscope(c.scope, key).Counter(metrics.CacheDeleteAttempt).Inc(1)
+	metrics.CacheDeleteRequestSubscope(c.scope, key).Counter(metrics.CacheDeleteAttempt).Inc(1)
```

Comment:
Changed the metrics for `DeleteRequest` because I think it was a mistake to use `CacheAddRequestSubscope `